src/person/person.repository.ts

```diff
@@ -19,7 +19,15 @@ export class PersonRepository extends RepositoryBase<IPerson> {
    */
   getPersonsByQuery(queryFields: any = {}, populate?: any, select?: any): Promise<IPerson[]> {
     const cond = {};
-    if (!(queryFields.alsoDead && queryFields.alsoDead === 'true')) cond['alive'] = 'true';
+
+    let query;
+    if (queryFields.status) {
+      query = queryFields.status.toLowerCase();
+    }
```

Comment:
can be:
`const query = queryFields.status ? queryFields.status.toLowerCase() : ''`